.circleci/config.yml

```diff
@@ -41,6 +41,11 @@ jobs:
           # Excludes tests that require Yale VPN access
           command:  |
             bundle exec rspec --tag ~vpn_only:true
+          environment:
```

Comment:
Since these are being set explicitly in the test, I think it will only cause confusion to have them here.

Reply:
I removed env REGISTRY_HOST but I'm pretty sure we still need to set the other 3 here otherwise they will appear blank when the pipeline is running the specs

Reply:
@orangewolf can you weight in based on your proposed changes to docker build and CI processes?  I definitely would not expect to see these values set here (and wouldn't think to look here without a lot of sleuthing)

Is there any way we can use the values in docker-compose.yml, .env, or .env.test in preference to putting them in the CircleCI config?

Reply:
We just need the code change below to allow the env vars to be blank w/o blowing up the code. This is good practice anyway so I think we can punt on the coming discussion about where env vars should be set so that they are not repeated so often.

Reply:
@maxkadel @mark-dce @orangewolf 
The ENV variables have been removed from .circleci/config.yml.